kernel/xpmem_mmu_notifier.c

```diff
@@ -17,7 +17,6 @@
 #include <linux/cdev.h>
 #include <linux/percpu.h>
 
-#include <asm/tlb.h>
```

Comment:
@alex--m you have to ifdef this include for versions below 5.10.x. For an example grep for LINUX_VERSION_CODE in the code